pages/common/hub.md

```diff
@@ -7,10 +7,18 @@
 
 `hub clone {{repo_name}}`
 
-- Clone another user repository:
+- Clone another user's repository:
 
 `hub clone {{github_username}}/{{repo_name}}`
 
+- Fork your own copy of a repository cloned from another user (creates a remote named after your GitHub username):
```

Comment:
Might be good to mention that you need to be in the directory of the repo already cloned.